packages/arb-token-bridge-ui/tests/support/index.ts

```diff
@@ -15,6 +15,19 @@ Cypress.Keyboard.defaults({
 
 logCollector()
 
+// Hide fetch/XHR requests
+const app = window.top
```

Comment:
I find all the RPC and fetch/XHR calls being logged in the Cypress window very annoying. They remove the focus from which test is running, so I decided to hide them. Wdyt?

Reply:
I would reuse something like this instead: https://github.com/OffchainLabs/arbitrum-token-bridge/pull/1439/files#diff-bf082afaa3cb06c491120e9dcb8770adb16cd9f788d5857164a964c3a5e2154dR16-R28